pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
```

Comment:
@jlowdermilk This should probably be AllAlpha to follow API const guidelines.  Object to me asking @girishkalele to fix it ?